WiserTaskScheduler/WiserTaskScheduler/Modules/Branches/Services/BranchQueueService.cs

```diff
@@ -2533,6 +2547,55 @@ private static async Task UpdateProgressInQueue(IDatabaseConnection databaseConn
             await databaseConnection.ExecuteAsync($"UPDATE {WiserTableNames.WiserBranchesQueue} SET items_processed = ?itemsProcessed WHERE id = ?queueId");
         }
 
+        private static async Task ReplaceStyledOutputIdInsideStyledOutput(IDatabaseConnection databaseConnection,
+            ulong targetStyledOutput, ulong oldId, ulong newId)
+        {
+            var originalDatabase = databaseConnection.ConnectedDatabase;
+
+            // Get current Content
+            var items = await databaseConnection.GetAsync($"""
+                                                           SELECT
+                                                           `format_begin`, `format_item`, `format_end`, `format_empty`
+                                                           FROM `{originalDatabase}`.`{WiserTableNames.WiserStyledOutput}` AS prod
+                                                           WHERE prod.id = {targetStyledOutput}
+                                                           ORDER BY file.id ASC
+                                                           LIMIT 1
+                                                           """);
+            if (items.Rows.Count == 0)
+            {
+                // The Target styledOutput does not exist, no need to update this one.
+                return;
+            }
+
+            var formatBegin = items.Rows[0].Field<string>("format_begin");
+            var formatItem = items.Rows[0].Field<string>("format_item");
+            var formatEnd = items.Rows[0].Field<string>("format_end");
+            var formatEmpty = items.Rows[0].Field<string>("format_empty");
+
+            // Note: Curly Brace needs to be included in search/replace
+            var searchString = "{StyledOutput~" + oldId.ToString();
+            var replaceString = "{StyledOutput~" + newId.ToString();
+
+            formatBegin = formatBegin.Replace(searchString, replaceString);
+            formatItem = formatItem.Replace(searchString, replaceString);
+            formatEnd = formatEnd.Replace(searchString, replaceString);
+            formatEmpty = formatEmpty.Replace(searchString, replaceString);
+
+            await databaseConnection.ExecuteAsync($"""
+                                                   UPDATE {WiserTableNames.WiserStyledOutput}
+                                                   SET
+                                                        format_begin = {formatBegin},
+                                                        format_item = {formatItem},
+                                                        format_end = {formatEnd},
+                                                        format_empty = {formatEmpty}
+                                                   WHERE id = {targetStyledOutput}
+                                                   ORDER BY file.id ASC
+                                                   LIMIT 1
+                                                   """);
+
```

Comment:
Please remove this empty line. It serves no purpose.

Reply:
correct,